after/syntax/typescript.vim

```diff
@@ -2,6 +2,8 @@ if dracula#should_abort('typescript')
     finish
 endif
 
+" HerringtonDarkholme/yats.vim {{{
```

Comment:
Are these groups defined by this plugin? Just to clarify

Reply:
Yes. Figured I'd match the format we have going in the `javascript.vim` file.

Reply:
Ah, gotcha. I don't use these filetypes (or have the patches to vim yet), but it looks fine to me.

Reply:
Yeah, vim is a slow moving ship when it comes to them actually pushing releases. This merged to master back in August!

Cool, I'll merge this in here then. :+1: